### PR TITLE
SimilarSites - Replaced IDs with classes (fixes #1691)

### LINKED ISSUE #1691: SimilarSites: Move away from using IDs
We often use classes in IAs and rarely IDs. Reference DOM using classes instead. 

https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/similar_sites/content.handlebars#L10

### PR DESCRIPTION
**Pull request by tagawa**
Also removed duplicate hide/expand (+/-) icons.

### CODE REVIEW

File: share/spice/similar_sites/similar_sites.css [hunk 1]
```diff
@@ -1,20 +1,16 @@
-.zci--similar_sites #hidden {
+.zci--similar_sites .hidden, .zci--similar_sites .chomp--link__ls {
```
Comment (moollaza) [inline share/spice/similar_sites/similar_sites.css]: I think we may already have `.hidden` set to `display: none` in our base CSS?

Comment (tagawa) [inline share/spice/similar_sites/similar_sites.css]: There's no ```.hidden``` in the current versions of <a href="https://duckduckgo.com/s956.css">s956.css</a> or <a href="https://duckduckgo.com/t956.css">t956.css</a>. Maybe in a to-be-released version?

Comment (moollaza) [inline share/spice/similar_sites/similar_sites.css]: sorry, our class name is actually, `is-hidden` -- you should be able to use that.

**Comment by tagawa** [comment]
As well as using ```.is-hidden```, was also able to remove a couple of JavaScript toggles thanks to existing behaviour with ```.is-expanded```.

**Comment by jagtalon** [comment]
:+1:

**Comment by tagawa** [comment]
Cool, thanks for the merge.

**Comment by jagtalon** [comment]
@tagawa no problem! I found there's a small bug that I missed but I made a fix in #1741

**Comment by tagawa** [comment]
Ah, well spotted. Thanks.

**Pull request merged**